.github/workflows/build_wheels.yml

```diff
@@ -189,9 +192,13 @@ jobs:
     needs:
       - build_source
       - check_env_variables
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, ubuntu-20.04]
     if: needs.check_env_variables.outputs.gcp-variables-set == 'true' && github.event_name != 'pull_request'
     steps:
+      - name: Install python
```

Comment:
should this (and below) be using the self-hosted-action?

Reply:
I was unsure at first because it required to add an extra checkout step for jobs that didn't include it, but probably unification is worth it, I've added it for remaining single installations, thanks @damccorm.